### PR TITLE
#1046: First iteration of using companies and standards

### PR DESCRIPTION
**Pull request by commjoen**
Thank you for submitting a Pull Request to the Mobile Security Testing Guide. Please make sure that:

- [ ] Your contribution is written in the 2nd person (e.g. you)
- [ ] Your contribution is written in an active present form for as much as possible.
- [ ] You have made sure that the reference section is up to date (e.g. please add sources you have used, make sure that the references to MITRE/MASVS/etc. are up to date)
- [ ] Your contribution has proper formatted markdown and/or code
- [ ] Any references to website have been formatted as [TEXT](URL “NAME”)
- [ ] You verified/tested the effectiveness of your contribution (e.g.: is the code really an effective remediation? Please verify it works!)

If your PR is related to an issue. Please end your PR test with the following line:
This PR covers issue #1046 .

### CODE REVIEW

**Comment by OMTGreleaser** [comment]
Results for commit dc5c18605760ab4691d5be5379059748bdba05f7: Broken link result: [!] Error(s) found by the Links validator: 2 pages have dead links! Verbose output in /link-check-result.out .
 Please run the Apply_Link_Check.sh in the tools folder to see wherhe the Link issues are. 
 Markdown result errorlength:        5 ../linter-result.out .
 Please run the Apply_Linter_check.sh in the tools folder to see where the Lint issues are.

**Comment by OMTGreleaser** [comment]
Results for commit a329ec6f121ea4aae3ae74ab623e5c5715a7af22: Broken link result: [!] Error(s) found by the Links validator: 3 pages have dead links! Verbose output in /link-check-result.out .
 Please run the Apply_Link_Check.sh in the tools folder to see wherhe the Link issues are. 
 Markdown result errorlength:        1 ../linter-result.out .
 Please run the Apply_Linter_check.sh in the tools folder to see where the Lint issues are.

File: Users.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+- [NVISO CVBA](https://www.nviso.be)
+- [Randorisec](https://randorisec.fr/)
+- [Secarma](https://www.secarma.com/)
+- [VantagePoint](https://www.vantagepoint.sg)
+- [Xebia](https://xebia.com)
+- [Websec Canada](https://www.websec.ca/mobile-application-security)
+
+## Standards, frameworks and guidelines recommending the MASVS and MSTG
+
+Currently, the MASVS and MSTG are recommending the MASVS and MSTG:
```
Comment (sushi2k) [inline Users.md]: Currently, the MASVS and MSTG are recommending the MASVS and MSTG: -> Currently, the MASVS and MSTG are recommended by:

**Comment by OMTGreleaser** [comment]
Results for commit 4d6da7cefd112a32abadc6808aeca4a296fbd4fa: Broken link result: [!] Error(s) found by the Links validator: 2 pages have dead links! Verbose output in /link-check-result.out .
 Please run the Apply_Link_Check.sh in the tools folder to see wherhe the Link issues are. 
 Markdown result errorlength:        1 ../linter-result.out .
 Please run the Apply_Linter_check.sh in the tools folder to see where the Lint issues are.

**Comment by OMTGreleaser** [comment]
Results for commit 34dd63dfa7cf8dbcad5ea0df7b59520fb5b454d9: Broken link result: [!] Error(s) found by the Links validator: 2 pages have dead links! Verbose output in /link-check-result.out .
 Please run the Apply_Link_Check.sh in the tools folder to see wherhe the Link issues are. 
 Markdown result errorlength:        1 ../linter-result.out .
 Please run the Apply_Linter_check.sh in the tools folder to see where the Lint issues are.

**Comment by OMTGreleaser** [comment]
Results for commit ae575cc1323384179ff0713fc7b370a274014dc9: Broken link result: [!] Error(s) found by the Links validator: 2 pages have dead links! Verbose output in /link-check-result.out .
 Please run the Apply_Link_Check.sh in the tools folder to see wherhe the Link issues are. 
 Markdown result errorlength:        1 ../linter-result.out .
 Please run the Apply_Linter_check.sh in the tools folder to see where the Lint issues are.

**Pull request merged**